### PR TITLE
Configure i18n

### PR DESCRIPTION
**Pull request by KozlovaAngelika**
Configure i18n

### CODE REVIEW

File: App.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  container: {
+    flex: 1,
+    alignItems: 'center',
+    justifyContent: 'flex-start',
+    paddingTop: 20,
+  },
+  text: {
+    color: '#808080',
+  },
+});
```
Comment (i-nemets) [inline App.tsx:20]: i think better:
 - create common constants for colors and fonstSizes 
 - move styles to separate file

File: __tests__/App-test.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import 'react-native';
+import React from 'react';
+import App from '../src/App';
+
+import renderer from 'react-test-renderer';
```
Comment (i-nemets) [inline __tests__/App-test.tsx:5]: I think need to remove empty line

File: package.json [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+        "start": "react-native start",
+        "test": "jest",
+        "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
+        "prepare": "husky install",
+        "precommit": "lint-staged"
+    },
+    "dependencies": {
+        "@material-ui/core": "^4.12.3",
+        "@types/history": "^4.7.3",
+        "i18next": "^20.3.5",
```
Comment (i-nemets) [inline package.json:17]: i think better remove version incrementing

File: package.json [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+            "js",
+            "jsx",
+            "json",
+            "node"
+        ]
+    },
+    "husky": {
+        "hooks": {
+            "pre-commit": "lint-staged"
+        }
```
Comment (i-nemets) [inline package.json:76]: I think lint-staged is not used and can be removed

File: src/styles.ts [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    flex: 1,
+    alignItems: 'center',
+    justifyContent: 'flex-start',
+    paddingTop: 20,
+  },
+  text: {
+    color: lightGray,
+  },
+});
+export default styles;
```
Comment (i-nemets) [inline src/styles.ts:18]: leave one empty line before export

File: src/utils/i18n/locales/en.json [hunk 6]
```diff
@@ -0,0 +1,4 @@
+{
+    "title": "testtitle",
+    "test": "asdas"
```
Comment (brusnev-egor) [inline src/utils/i18n/locales/en.json:3]: remove it

File: src/utils/i18n/locales/ru.json [hunk 7]
```diff
@@ -0,0 +1,4 @@
+{
+    "title": "тест",
+    "test": "фсдс"
```
Comment (brusnev-egor) [inline src/utils/i18n/locales/ru.json:3]: remove this line

**Review by brusnev-egor — COMMENTED** [review]
pull latest dev to this branch

File: src/App.tsx [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+  const { t } = useTranslation();
+  return (
+    <View style={styles.container}>
+      <Text style={styles.text}>
+        {t('title')}
+      </Text>
+    </View>
+  );
+};
+export default App;
```
Comment (i-nemets) [inline src/App.tsx:20]: leave one empty line before export

File: src/styles.ts [hunk 9]
```diff
@@ -0,0 +1,17 @@
+import { StyleSheet } from 'react-native';
+
+const lightGray = '#808080';
```
Comment (brusnev-egor) [inline src/styles.ts:3]: move to constants

**Review by brusnev-egor — APPROVED** [review]

**Review by i-nemets — APPROVED** [review]

**Pull request merged**